dynamodb/client.go

```diff
@@ -0,0 +1,95 @@
+package dynamodbclient
+
+import (
+	"github.com/aws/aws-sdk-go/aws"
+	"github.com/aws/aws-sdk-go/aws/session"
+	"github.com/aws/aws-sdk-go/service/dynamodb"
+	"github.com/aws/aws-sdk-go/service/dynamodb/dynamodbattribute"
+	"github.com/aws/aws-sdk-go/service/dynamodb/dynamodbiface"
+)
+
+const (
+	TableHashKey = "conceptId"
+)
+
+type Model struct {
+	UUID         string   `json:"conceptId"`
+	ConcordedIds []string `json:"concordedIds"`
+}
+
+type DynamoDbClient interface {
+	Read(uuid string) (Model, error)
+	Write(m Model) (Model, error)
+	Delete(uuid string) (Model, error)
+	Count() (int64, error)
+}
+
+type DynamoDbClientImpl struct {
```

Comment:
I would have `type DynamoClient struct`.  Impl is a hangover from Java and isn't very Go.